src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "caseSensitive": true,
   "completion": {
-    "useContextAwareSorting": true
+    "useContextAwareSorting": true,
+    "useTemplateForInvocationOnlySnippet": false
```

Comment:
I updated all of the templates to disable editor templates for invocation-only snippets similar to how I enabled context-aware sorting in the templates.